### PR TITLE
Fix for classSet and board warnings

### PR DESCRIPTION
**Pull request by nikolauska**
React.addons.classSet is going to be deprecated and this will replace it with JedWatson/classnames as pointed here https://facebook.github.io/react/docs/class-name-manipulation.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +15,7 @@
   },
   "dependencies": {
     "babel": "5.1.10",
+    "classnames": "^2.1.3",
```
Comment (melonmanchan) [inline package.json]: No hat

**Comment by tanelih** [comment]
???

**Comment by nikolauska** [comment]
Added description what this pull request does

**Comment by tanelih** [comment]
Good jobe :dog2:

**Pull request merged**